tests/pods_test.go

```diff
@@ -1,16 +1,113 @@
 package tests
 
 import (
+	"context"
+	"fmt"
+	"strings"
+
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/k8snetworkplumbingwg/kubemacpool/pkg/names"
 )
 
+const defaultNumberOfReplicas = 2
+
 var _ = Describe("Pods", func() {
-	Context("Check the client", func() {
-		It("should not fail", func() {
-			_, err := testClient.KubeClient.CoreV1().Pods("").List(v1.ListOptions{})
+	Context("Check the pod mutating webhook", func() {
+		AfterEach(func() {
+			for _, namespace := range []string{TestNamespace, OtherTestNamespace} {
+				podList := &corev1.PodList{}
```

Comment:
Please add inline comments explaining what are you cleaning up and why.

Reply:
done